core/examples/driver/main.cpp

```diff
@@ -75,12 +75,20 @@ int main(int /*argc*/, char** /*argv*/)
 {
   char cmd[256];
 
-  std::vector<std::string> availableModules = GetExampleModules();
+  FrameworkFactory factory;
+  Framework* framework = factory.newFramework(std::map<std::string, std::string>());
+  framework->init();
+  framework->Start();
```

Comment:
As Wayne asked somewhere else, is there a reason for separate `init` and `Start` calls? Can we get rid of the `init`?